llms/ollama/ollama_test.go

```diff
@@ -7,17 +7,61 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	tcollama "github.com/testcontainers/testcontainers-go/modules/ollama"
 	"github.com/tmc/langchaingo/llms"
 )
 
+const (
+	ollamaVersion  string = "0.3.13"
+	llamaModel     string = "llama3.2"
+	llamaTag       string = "1b" // the 1b model is the smallest model, that fits in CPUs instead of GPUs.
```

Comment:
llamaTag confused me with image tag.

```suggestion
	llamaModelTag       string = "1b" // the 1b model is the smallest model, that fits in CPUs instead of GPUs.
```

Reply:
Done in [af47c08](https://github.com/tmc/langchaingo/pull/1079/commits/af47c08720e17cb330ae1a3ff42c6632289ab174), thanks!